user/build-stages.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Build Stages
+permalink: /user/build-stages/
+layout: en
+---
+
+<div id="toc"></div>
+
+> Build stages are still in BETA. There is more information about what this means, and how you can give us feedback on this new feature in the [GitHub issue](https://github.com/travis-ci/beta-features/issues/11).
```

Comment:
Valid point. You *could* construe still as having been there a while, so maybe currently is a better choice. We use `still` on at least one of the other beta notices.